doc/tutorials/introduction/load_save_image/load_save_image.rst

```diff
@@ -28,6 +28,8 @@ Here it is:
 
    #include <cv.h>
    #include <highgui.h>
+   #include <opencv2/imgproc/imgproc.hpp>
+   #include <stdio.h>
```

Comment:
Actually I would suggest using
```#include "..."
```
instead of `<...>` for all the OpenCV headers and leaving `<stdio.h>`.
It allows using the user-provided OpenCV version and not only the system installed one.
@alalek , do you agree?